.changeset/perfect-fans-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/mdx': minor
+---
+
+Updates adapter server entrypoint to use `@astrojs/mdx/server.js`
```

Comment:
```suggestion
Updates adapter server entrypoint to use `@astrojs/mdx/server.js`

This is an internal change. Handling JSX in your `.mdx` files has been moved from Astro internals and is now the responsibility of this integration. You should not notice a change in your project, and no update to your code is required.
```
Wondering about a statement explaining that they shouldn't need to worry about this?

Reply:
Looks good to me. Doesn't hurt to cover the possible concerns!